clusterloader2/pkg/test/simple_test_executor.go

```diff
@@ -86,8 +86,12 @@ func (ste *simpleTestExecutor) ExecuteTest(ctx Context, conf *api.Config) *error
 		if ctx.GetClusterLoaderConfig().ReportDir == "" {
 			klog.Infof("%v: %v", summary.SummaryName(), summary.SummaryContent())
 		} else {
+			id := ""
+			if ctx.GetClusterLoaderConfig().TestScenario.Identifier != "" {
+				id = "_" + ctx.GetClusterLoaderConfig().TestScenario.Identifier
```

Comment:
Why the id starts with "_" ?

Reply:
I'm using it later in file name. I want to add additional separator "_" if id is not empty.
When identifier provided: SummaryName_configName_testId_timstamp.ext.
Without id: SummaryName_configName_timstamp.ext

Reply:
well, id is not the most intuitive name then :)

infix? distinctor? (neither of those is good, but less confusing)

Reply:
Changed to "testDistinctor".